app/templates/electron/main-process/electron-main.ts

```diff
@@ -0,0 +1,58 @@
+import { app, BrowserWindow, nativeTheme } from 'electron'
+
+try {
+  if (process.platform === 'win32' && nativeTheme.shouldUseDarkColors === true) {
+    require('fs').unlinkSync(require('path').join(app.getPath('userData'), 'DevTools Extensions'))
+  }
+} catch (_) { }
+
+/**
+ * Set `__statics` path to static files in production;
+ * The reason we are setting it here is that the path needs to be evaluated at runtime
+ */
+if (process.env.PROD) {
+  //@ts-ignore
```

Comment:
`@ts-ignore` isn't really an acceptable way to fix TS issues :P
Which error was it throwing previously? Maybe TS miss the context to understand this file is run into Node environment and must be made aware of

Reply:
The error is that `__statics` is not defined on `Global`. This means the environment(vscode/tsc) know that this file is to run into Node environment. I think this can be handled using .d.ts declaration file. I still new to typescript. I am learning on it.

Reply:
No problem, contributing is a good way to learn :P
I think a .d.ts declaration which augment `Global` interface should do the trick